docs/website/docs/v1.x/dsl/socket/socket.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Socket"
+sidebar_label: "Socket"
+---
+
+`Socket[-R, +E, -A, +B]` models a function from `A` to `ZStream[R, E, B]`. When a value of type `A` is evaluated against a `Socket[R, E, A, B]`, it can either succeed with a `B`, or fail with an `E`, and it could have its requirement on `R`.
```

Comment:
```suggestion
`Socket[-R, +E, -A, +B]` models a function from `A` to `ZStream[R, E, B]`. When a value of type `A` is evaluated against a `Socket[R, E, A, B]`, it can either succeed with a stream of values of type `B`, or fail with an `E`, and it could have its requirement on `R`.
```